packages/core/useBreakpoints/breakpoints.ts

```diff
@@ -26,15 +26,17 @@ export const breakpointsBootstrapV5 = {
 }
 
 /**
- * Breakpoints from Vuetify V2
+ * Breakpoints from Vuetify V3
  *
- * @see https://vuetifyjs.com/en/features/breakpoints
+ * @see https://vuetifyjs.com/en/styles/float/#overview
  */
 export const breakpointsVuetify = {
-  xs: 600,
-  sm: 960,
-  md: 1264,
-  lg: 1904,
+  xs: 0,
```

Comment:
What about to add a new preset `breakpointsVuetifyV3` instead of replacing the old one. That might cause ppl update and found behavior changed

Reply:
Should it be breakpointsVuetify and breakpointsVuetifyV3 
or breakpointsVuetifyV2  and breakpointsVuetifyV3 ?

Reply:
breakpointsVuetify and breakpointsVuetifyV3

Reply:
we are not supposed to change the exist one